src/scitokens/tools/verify_token.py

```diff
@@ -0,0 +1,61 @@
+"""
+Verify a SciToken from command-line inputs.
+"""
+
+import argparse
+import scitokens
+from scitokens.utils.errors import InvalidTokenFormat
+from scitokens.utils.errors import MissingIssuerException
+from scitokens.utils.errors import MissingKeyException
+from scitokens.utils.errors import NonHTTPSIssuer
+from scitokens.utils.errors import SciTokensException
+from scitokens.utils.errors import UnableToCreateCache
+from scitokens.utils.errors import UnsupportedKeyException
+
+
+def add_args():
+    """
+    Generate the ArgumentParser object for the CLI.
+    """
+    parser = argparse.ArgumentParser(description='Verify a new SciToken')
+    parser.add_argument('--token', help='The serialized string of SciToken')
+    parser.add_argument('-v', '--verbose', action='store_true')
+
+    args = parser.parse_args()
+    return args
+
+
+def main():
+    """
+    Given a serialized SciToken, verify it and return an error message in case of failure.
+    """
+    args = add_args()
+    stoken = None
+    try:
+        stoken = scitokens.SciToken.deserialize(args.token)
```

Comment:
```suggestion
        stoken = scitokens.SciToken.deserialize(args.token[0])
```
When you specify nargs, it puts the arguments in a list, so just grab the first.